README.md

```diff
@@ -132,6 +138,23 @@ _           | `host.py`                   | (to be removed) control a remote dev
 **others**  | `launch.sh`                 | shell script to start `pvm.py` with one _click_
 _           | `build_omxplayer.sh`                 | shell script to build `omxplayer` with one _click_
 
+## How to deploy your code when you testing in multiple rpis?
+Use the [deploy_code_to_rpi.sh](https://github.com/omarcostahamido/PVM/blob/AddCDscript/deploy_code_to_rpi.sh) script.
+But there are two prerequisites:
+
+One is the need to set up passwordless ssh access.
+For the passwordless ssh access please follow [here](https://danidudas.medium.com/how-to-connect-to-raspberry-pi-via-ssh-without-password-using-ssh-keys-3abd782688a).
```

Comment:
ah ok, it is using private-public rsa keys. That's much better! Thanks.
maybe you should change the text slightly to clarify that the goal is not simply to remove the password, it is also to impose a rsa public-private key.
I've saved this article (nice one) on [wayback machine](https://web.archive.org/web/20221029175737/https://danidudas.medium.com/how-to-connect-to-raspberry-pi-via-ssh-without-password-using-ssh-keys-3abd782688a).

Reply:
Now I hope this script will alleviate our deployment problems during development testing. This way the secret key form will reduce the time we spend on testing.
If there are future deployment issues during production, perhaps a cycle update approach could be considered.